inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -456,122 +430,130 @@ population_data_age_months <- gen_population(total_pop = 5000,
-## When you copy and paste file paths, you need to change \ to /.
-##
 
-## Excel file
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+lab_data
+
+If you have lab data, you can read it in here.
+You will need to make sure that your identifier (e.g. case_number) matches the
+linelist.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+```{r lab_data}
+## Read data -------------------------------------------------------------------
+## This step reads in your lab data from Excel.
+## You may need to rename your columns.
+
 ## to read in a specific sheet use "which"
 # lab_data <- rio::import(here("labresults.xlsx"), which = "Sheet1")
 
-## repeat same cleaning steps as in standardise_clean_data code chunk as appropriate
+## repeat preparation steps as appropriate
 ## make sure your unique ID variable name matches!
 
-## Fake data ------------------------------------ 
-## THE BELOW CAN BE DELETED IF YOU HAVE YOUR OWN (REAL) LABORATORY DATA! 
-## its just to be able to demonstrate script in this template
-
-# # generate artificial lab tests results
-# lab_results <- linelist_cleaned %>% 
-#                 select(case_number) %>% 
-#                 mutate(test_result = sample(c("Positive", "Negative"),
-#                                             nrow(linelist_cleaned), 
-#                                             replace = TRUE)
-#                        )
-# 
-# # merging linelist with lab dataset 
+## Merge lab to linelist -------------------------------------------------------
 # linelist_cleaned <- left_join(linelist_cleaned, lab_results, 
 #                               by = "case_number")
 ```
 
 
 
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+browse_data
 
+You'll want to look at your data. Here are a few ways you can explore.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
 ```{r browse_data, eval = FALSE}
-## Browsing data ---------------------------------
-## here are a few ways to do data explorations 
-
 ## view the first ten rows of data
 head(linelist_cleaned, n = 10)
 
-## view your whole dataset interactivley (in an excel style format)
-## Remember that `View` needs to be written with a capital *V*
+## view your whole dataset interactively (in an excel style format)
 View(linelist_cleaned)
 
 ## overview of variable types and contents
 str(linelist_cleaned)
 
-## gives mean, median and max values of variables
-## gives counts for categorical variables
+## get summary: 
+## mean, median and max values of numeric variables
+## counts for categorical variables
 ## also gives number of NAs
 summary(linelist_cleaned)
 
 ## view unique values contained in variables 
-unique(linelist_cleaned$sex)
+## you can run this for any column -- just replace the column name
+unique(linelist_cleaned$patient_origin) 
 
-## another alternative is with the "summarytools package"
```

Comment:
whats reasoning for deleting this?

Reply:
I dont see the issue in having it here if the chunk is eval = FALSE anyway

Reply:
I don't understand why we want to drop in summarytools without explanation -- there are many alternatives for viewing your data. It's not clear why we want to throw summarytools at people here without any idea of why it's helpful or what it would do differently. At best, this line will be ignored. At worst, people waste a ton of time trying to figure out a new package that we don't explain or otherwise use.

Reply:
its a one liner that gives you an easy overview, dont think there is that much room for confusion.... but fine to delete.